src/plonk/circuit.rs

```diff
@@ -33,6 +39,14 @@ pub trait Assignment<F: Field> {
         to: impl FnOnce() -> Result<F, Error>,
     ) -> Result<(), Error>;
 
+    /// Assign an auxiliary wire value (external)
+    fn assign_aux(
```

Comment:
We don't need a way to assign the aux wire values from the prover side; those are fixed outside of the prover. Their values should be supplied as part of the call to `Proof::create`, and the commitments should be supplied as part of the call to `Proof::verify`.